### DISCUSSION TITLE
Version 6.0.2.0

### DISCUSSION THREAD

**Discussion by rsjaffe** [rsjaffe/MIDI2LR#775] [Announcements]
## What's Changed
* Fix problems with knobs values restore by @AndyMcFishka in https://github.com/rsjaffe/MIDI2LR/pull/771

## New Contributors
* @AndyMcFishka made their first contribution in https://github.com/rsjaffe/MIDI2LR/pull/771

**Full Changelog**: https://github.com/rsjaffe/MIDI2LR/compare/v5.4.0.0...v6.0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/rsjaffe/MIDI2LR/releases/tag/v6.0.2.0'>Version 6.0.2.0</a>.</em>

**Comment by FujiGuy** [comment]
The installer is not working on my pc.

**Comment by antzpantz** [comment]
I've not had any troubles installing it...

**Comment by FujiGuy** [comment]
It's working now, I disabled the F-Secure Deepguard protection for the install. So it was a problem on my side.

**Comment by antzpantz** [comment]
Good to hear! 🙏
